spec/components/review_draft/draft/view_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe ReviewDraft::Draft::View do
+  alias_method :component, :page
+
+  before do
+    render_inline(described_class.new(trainee: trainee))
+  end
+
+  context "sections that appear for all trainees regardless of route" do
+    let(:trainee) { create(:trainee) }
+
+    it "shows for all trainees" do
+      expect(component).to have_text("Personal details")
+      expect(component).to have_text("Contact details")
+      expect(component).to have_text("Diversity information")
+      expect(component).to have_text("Degree")
+      expect(component).to have_text("Course details")
+      expect(component).to have_text("Trainee start date and ID")
+    end
+  end
+
+  context "when the trainee is on the provider-led route", 'feature_routes.provider_led_postgrad': true do
+    let(:trainee) { create(:trainee, :provider_led_postgrad, :with_placement_assignment) }
+
+    it "shows provider-led specific sections" do
+      expect(component).to have_text("Placement details")
+    end
+  end
+
+  context "when the trainee is on the school direct routes", 'feature_routes.school_direct_salaried': true do
```

Comment:
Could we test for both school direct routes here?

something like:

```ruby
%i[school_direct_salaried school_direct_tuition_fee].each do |route|
  context "when the trainee is on the #{route} routes", 'feature_routes.#{route}': true do
    let(:trainee) { create(:trainee, route) }

    it 'blah'
      'test'
    end
  end
end
```

Reply:
Thanks, I've added